contact_map/concurrence.py

```diff
@@ -63,6 +65,41 @@ def __getitem__(self, label):
 
         # return sum(coincidence_list) / np.sqrt(norm_sq)
 
+def _regularize_contact_input(contact_input, atom_or_res):
+    """Clean input for concurrence objects.
+
+    The allowed inputs are the :class:`.ContactFrequency`, or the
+    :class:`.ContactObject` coming from the ``.atom_contacts`` or
+    ``.residue_contacts`` attribute of the contact frequency, or the list
+    coming from the ``.most_common()`` method for the
+    :class:`.ContactObject`.
+
+    Parameters
+    ----------
+    contact_input : many possible types; see method description
+        input to the contact concurrences
+    atom_or_res : string
+        whether to treat this as an atom-based or residue-based contact;
+        allowed values are "atom", "res", and "residue"
+
+    Returns
+    -------
+    list :
+        list in the format of ``ContactCount.most_common()``
+    """
+    if isinstance(contact_input, contact_map.ContactFrequency):
```

Comment:
shouldn't this be `isinstance(contact_input, contact_map.ContactObject)` ? As this statement would be `False` for a `ContactMap` or `ContactDifference`